monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/JobInvocationController.java

```diff
@@ -184,6 +153,90 @@ public String eval(String columnPath) {
         return output;
     }
 
+    /**
+     * Takes a JSON request body in this form (2 rows with 1 int and 2 strings
+     * each):
+     *
```

Comment:
This comment is not so clear: Should be something like: takes a JSon request body containing an array of key value pairs. and then the example.